brands/amenity/bank.json

```
@@ -6122,6 +6122,21 @@
       "name:ru": "Альфа-Банк"
     }
   },
+  "amenity/bank|Альфа-Банк~Украина": {
```

We add parenthesis for these, it helps to make them more readable in a list.
```suggestion
  "amenity/bank|Альфа-Банк~(Украина)": {
```

---

And you'd also have to add a "~(something)" to the other entry.